src/components/Input.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Input({ 
+    type, 
+    placeHolder
+}) {
+    return (
+        <input 
+        className="insert--registration"
+        type={type}
+        placeholder={placeHolder}
+        >
+        </input>
```

Comment:
Eu não sei se vc leu direito, mas vc já viu algum input ser chamdo da forma que vc escreveu? Inputs não tem tag de fechamento, é tag unica

```suggestion
        />
```